tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -123,6 +125,90 @@ public static OpenSpan startSpan(String operation) {
         return startSpanInternal(operation, SpanType.LOCAL);
     }
 
+    /**
+     * Like {@link #startSpan(String, SpanType)}, but does not set or modify tracing thread state.
+     */
+    public static DetachedSpan startDetachedSpan(String operation, SpanType type) {
```

Comment:
This might be clearer if we call it `startAsyncSpan` instead.

Reply:
However it's not necessarily asynchronous, it's no different from any other span except that it is not connected to any thread-local state.